magento/magento2
# Check if OpenGraph can show price

### Description (*)
This checks if the price can be shown


### Manual testing scenarios (*)
- Create a plugin which hides the price, or use https://github.com/karliuka/m2.ProductAvailable
- Check that the price is hidden
- Check that without this change, the price is still visible in the OG tag.


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#38512: Check if OpenGraph can show price
